pkg/assembler/graphql/schema/certifyVEXStatement.graphql

```diff
@@ -52,6 +52,8 @@ type CertifyVEXStatement {
   subject: PackageOrArtifact!
   "Attested vulnerability"
   vulnerability: Vulnerability!
+  "status of the vulnerabilities with respect to the products and components listed in the statement"
+  status: String!
```

Comment:
Should we make this an enum, since there's a VEX spec that says what values this can have?